llzk_backend/Cargo.toml

```diff
@@ -8,7 +8,13 @@ edition = "2018"
 program_structure = { path = "../program_structure" }
 ansi_term = "0.12.1"
 anyhow = "1"
+num-bigint-dig = "0.8.4"
+num-traits = "0.2.6"
 mlir-sys = "0.5.0"
 melior = "0.25.0"
 melior-macro = "0.18.0"
 llzk = { git = "https://github.com/Veridise/llzk-rs", package = "llzk" }
+
+[patch.crates-io]
+# This makes the WalkOrder and WalkResult APIs public.
+melior = { git = "https://github.com/0xddom/melior.git", branch = "main" }
```

Comment:
It's already merged and I have a PR for updating the dependency [here](https://github.com/Veridise/llzk-rs/pull/52)